packages/jest-console/README.md

```diff
@@ -0,0 +1,90 @@
+# @wordpress/jest-console
+
+Custom [Jest](http://facebook.github.io/jest/) matchers for the [Console](https://developer.mozilla.org/en-US/docs/Web/API/Console)
+object to test JavaScript code in WordPress.
+
+This package converts `console.error` and `console.warn` functions into mocks and tracks their calls.
+It also enforces usage of one of the related matchers whenever tested code calls one of the mentioned `console` methods.
+It means that you need to assert with `.toHaveErrored()` or `.toHaveErroredWith( arg1, arg2, ... )` when `console.error`
+gets executed, and `.toHaveWarned()` or `.toHaveWarnedWith( arg1, arg2, ... )` when `console.warn` is called.
+Your test will fail otherwise! This is a conscious design decision which helps to detect deprecation warnings when
+upgrading dependent libraries or smaller errors when refactoring code.
+
+## Installation
+
+Install the module:
+
+```bash
+npm install @wordpress/jest-console --save-dev
+```
+
+### Setup
+
+The simplest setup is to use Jest's `setupTestFrameworkScriptFile` config option:
+
+```js
+"jest": {
+  "setupTestFrameworkScriptFile": "./node_modules/@wordpress/jest-console/build/index.js"
+},
+```
+
+If your project already have a script file which setups test framework, you will need add the following import statement:
```

Comment:
Minor: Grammar: "setups" -> "sets up"

Reply:
Alos, `has` instead of `have` please, "If your project has a script file...."

Reply:
Updated, thanks for corrections.